src/extension/store.ts

```diff
@@ -17,8 +18,14 @@ export class Store {
     @observable resultsFixed = [] as string[] // JSON string of ResultId. TODO: Migrate to set.
     @computed get results() {
         const runs = this.logs.map(log => log.runs).flat();
-        return runs.map(run => run.results ?? []).flat()
+        const filteredResults = runs.map(run => run.results ?? []).flat()
             .filter(result => !this.resultsFixed.includes(JSON.stringify(result._id)));
```

Comment:
Nitpick: You can use `.flatMap` here or even go for a `.reduce` to do everything in one iteration

Reply:
Agree, point taken 👍

This is the code which was there already (SARIF ext source code) and I just changed it assigning result to var here. I try to modify as little as needed here (in case of e.g. needing to easily merge upstream changes) so I will leave it as is.